Subnet-Calculator/admin/sw.js

```
@@ -0,0 +1,42 @@
+// navigator.serviceWorker.register('sw.js') call. On admin pages that
+// URL resolved to /admin/sw.js, which did not exist — the browser
+// parked the registration in a permanent "trying to install" state
+// (404 on the script fetch never lets install complete) and DevTools
+// could not unregister it because there was no installed worker yet.
+//
+// This file exists solely to let those stuck registrations finally
+// install — and immediately unregister themselves so the admin scope
+// is left clean. New admin sessions never register this file, because
+// app.js (v3.2.3+) skips registration on /admin/* paths.
+
+self.addEventListener('install', event => {
```

ESLint

[error] 'self' is not defined.

(no-undef)